k8s/jenkins/deployer/Dockerfile

```diff
@@ -1,21 +1,39 @@
 ARG MARKETPLACE_TOOLS_TAG
-FROM gcr.io/cloud-marketplace-tools/k8s/deployer_envsubst:$MARKETPLACE_TOOLS_TAG
+FROM launcher.gcr.io/google/debian9 AS build
 
-COPY manifest/* /data/manifest/
-COPY schema.yaml /data/
-COPY apptest/deployer /data-test/
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends gettext
+
+ADD chart/jenkins /tmp/chart
+RUN cd /tmp && tar -czvf /tmp/jenkins.tar.gz chart
+
+ADD apptest/deployer/jenkins /tmp/test/chart
+RUN cd /tmp/test \
+    && tar -czvf /tmp/test/jenkins.tar.gz chart/
+
+ADD schema.yaml /tmp/schema.yaml
 
-# Provide registry prefix and tag for default values for images.
 ARG REGISTRY
 ARG TAG
 
 ENV WAIT_FOR_READY_TIMEOUT 1800
 ENV TESTER_TIMEOUT 1800
 
-RUN cat /data/schema.yaml \
+RUN cat /tmp/schema.yaml \
     | env -i "REGISTRY=$REGISTRY" "TAG=$TAG" envsubst \
-    > /data/schema.yaml.new \
-    && mv /data/schema.yaml.new /data/schema.yaml
+    > /tmp/schema.yaml.new \
+    && mv /tmp/schema.yaml.new /tmp/schema.yaml
+
+
+FROM gcr.io/cloud-marketplace-tools/k8s/deployer_helm:$MARKETPLACE_TOOLS_TAG
+
+ARG REGISTRY
+ARG TAG
+
+COPY --from=build /tmp/jenkins.tar.gz /data/chart/
```

Comment:
Could you wait with submitting this PR till https://github.com/GoogleCloudPlatform/click-to-deploy/pull/423 is not submitted? This deployer is not correctly merged with @vcanaa change from https://github.com/GoogleCloudPlatform/click-to-deploy/pull/448.

I've fixed this in https://github.com/GoogleCloudPlatform/click-to-deploy/pull/423/commits/feba7617ea0496486603197917a71b7b3fe16b38 and will submit https://github.com/GoogleCloudPlatform/click-to-deploy/pull/423 once verification pipeline pass.

Reply:
I've successfully submitted the PR. Thank you!

Reply:
I'll send a PR to your branch that fix the conflicts.

Reply:
https://github.com/GoogleCloudPlatform/click-to-deploy/pull/458